cleartk-berkeleyparser/src/test/java/org/cleartk/berkeleyparser/ParserAnnotatorTest.java

```diff
@@ -102,6 +110,110 @@ public void test() throws UIMAException {
 
   }
 
+
+  @Test
+  public void givenASentenceWithoutPosesWhenParsingThenSyntaxTreeOfTheSentenceIsConstructed() throws ResourceInitializationException, AnalysisEngineProcessException{
+    jCas.setDocumentText(SAMPLE_SENT);
+    new Sentence(jCas, 0 , SAMPLE_SENT.length()).addToIndexes();
+
+    SimplePipeline.runPipeline(jCas, DefaultBerkeleyTokenizer.getDescription(), 
+        ParserAnnotator.getDescription(MODEL_PATH), 
+        ParseTreePosTagSetter.getDescription());
```

Comment:
This seems wrong.  Why are POS tokens getting set after running the parser?  The parser requires POS tags to run.